src/transformState.js

```diff
@@ -5,7 +5,30 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
-}
+  const add = 'addProperties';
+  const remove = 'removeProperties';
+  const clearProp = 'clear';
+
+  for (let i = 0; i < actions.length; i++) {
```

Comment:
You can just do
```js
actions.forEach(action => {
```
instead of 
```js
for (let i = 0; i < actions.length; i++) {
  const action = actions[i]
```